packages/closer/pages/events/[slug]/report.tsx

```
@@ -0,0 +1,330 @@
+  const ticketsByType = Array.from(ticketTypeMap.entries()).map(([name, data]) => ({
+    name,
+    value: data.count,
+    revenue: data.revenue
+  }));
+
+  // Calculate total booking revenue
+  let bookingRevenue = 0;
+  bookings && bookings.forEach((booking: any) => {
+    const total = booking.getIn(['total', 'val']) || 0;
+    bookingRevenue += total;
+  });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)